src/main/java/org/jabref/model/entry/BibEntry.java

```diff
@@ -622,6 +622,15 @@ public String getAuthorTitleYear(int maxCharacters) {
     }
 
     /**
+     * Returns the title of the given BibTeX entry as an optional.
+     *
+     * @return the title of a BibTeX entry or null if no title was found.
+     */
+    public Optional<String> getTitle() {
+        return getField(FieldName.TITLE);
```

Comment:
I've based the getter for the title on the other present getters in the `BibEntry` class. 
All these getters also return an empty `Optional` in case one isn't found and leave the error handling to be performed by the methods using it.

I can change it to return an empty string, if that seems to be the better option.